### DISCUSSION TITLE
Migrating from cakephp 3.x to cakephp 4.x

### DISCUSSION THREAD

**Discussion by imeneoh** [MayMeow/cakephp-fileupload#12] [Q&A] [answered]
Hello I need Some Help , I Have a CakePhP Project in version 3.x I want To migrate in to 4.x is it possible and can anyone Help what are the steps to follow to migrate from 3.x to 4.x CakePhp version

**Comment by maymeow** [comment, marked as answer]
Hello!

Here you can check their migration guide: https://book.cakephp.org/4/en/appendices/4-0-upgrade-guide.html
